patches/extensions-common-api-_permission_features.json.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/extensions/common/api/_permission_features.json b/extensions/common/api/_permission_features.json
+index c89ba3f84c35ed0dff9bb26fb581edbfbff6f90c..da57157cba93500b08e2a79e4c0a486c836805cc 100644
+--- a/extensions/common/api/_permission_features.json
++++ b/extensions/common/api/_permission_features.json
+@@ -328,6 +328,7 @@
+       "channel": "stable",
+       "extension_types": ["extension"],
+       "allowlist": [
++        "21070F3D60711361C1210B870439BE49B5D995F4",  // Ethereum Remote Client extension
```

Comment:
this should have gone in https://github.com/brave/brave-core/blob/master/common/extensions/api/_permission_features.json or better yet we should dump the existing patch in feature_compiler.py and do a chromium_src override for `AddCorePermissionFeatures` in the permission_features.h generated file